src/dcos_e2e/backends/_docker/__init__.py

```diff
@@ -157,6 +158,13 @@ def __init__(
                 custom network on macOS.
                 Therefore, it is recommended that you use this in conjunction
                 with the ``transport`` parameter.
+            one_master_host_port_map: The exposed host ports for one of the
```

Comment:
"one of the master node" > "one of the master nodes".

I'm guessing that "OSX" will cause a spellcheck error - "macOS" will not.

Reply:
Fixed.